worker/runtime/integration/integration_test.go

```diff
@@ -38,9 +36,10 @@ func (s *IntegrationSuite) containerdSocket() string {
 }
 
 func (s *IntegrationSuite) startContainerd() {
-	command := exec.Command("/usr/local/concourse/bin/containerd",
+	command := exec.Command("containerd",
```

Comment:
I remember this was significant because the `concourse/dev` image has `docker` installed, which has its own version of `containerd` -- if you just trust the `PATH` I think you end up running that one, and that is bad for reasons I don't remember. I had some extended chats (that I apparently didn't fully grok) with @cirocosta on the topic.

Reply:
Hm that's an interesting point. FWIW I went into `concourse/dev`, and the default `containerd` path is pointing to the one packaged with concourse.

```
~/w/concourse (master|✚1) $ docker run -it --entrypoint "/bin/bash" --privileged concourse/dev
root@cc594091cf1a:/go# which containerd
/usr/local/concourse/bin/containerd
```